src/commonMain/kotlin/com/algolia/search/model/settings/Settings.kt

```diff
@@ -357,11 +357,18 @@ public data class Settings(
     /**
      *  Lets you store custom data in your indices.
      */
-    @SerialName(KeyUserData) var userData: JsonObject? = null
+    @SerialName(KeyUserData) var userData: JsonObject? = null,
+    /**
+     * This parameter configures the segmentation of Japanese text at indexing time.
```

Comment:
Why not making it future proof by not tying it to japanese specifically? Considering the parameter name, it's expected more languages will be supported
```suggestion
     * This parameter configures the segmentation of text at indexing time.
```